roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -29,5 +29,10 @@
         "/etc/nvme:/etc/nvme",
         "/var/lib/openstack/config/ceph:/var/lib/kolla/config_files/ceph:ro",
         "/etc/ssh/ssh_known_hosts:/etc/ssh/ssh_known_hosts:ro"
+{% if edpm_nova_extra_bind_mounts | length > 0 %}
```

Comment:
Perhaps a nit, but is this test necessary? Just let L33 do its thing, which I believe will yield the same results even when the list is empty.

Reply:
i can try removing it 
i am default to the empty list so it may not be

Reply:
im testing this locally and ill push up a new verison once im sure this works